wire-ios/Wire-iOS Tests/BackupViewControllerTests.swift

```diff
@@ -17,38 +17,32 @@
 //
 
 import Foundation
-@testable import Wire
+import WireCommonComponents
 import XCTest
 
-final class BackupViewControllerTests: ZMSnapshotTestCase {
+@testable import Wire
+
+final class BackupViewControllerTests: XCTestCase {
```

Comment:
Great to see `ZMSnapshotTestCase` removed!

Reply:
:)